AISKU/README.md

```diff
@@ -184,6 +184,16 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 	npm run test
 	```
 
+## Performance
+At just 22 KB gzipped, and taking only ~15 ms to initialize, Application Insights adds a neglible amount of loadtime to your website. By using the snippet, the essential components of the script are loaded inline synchronously, so it does not require a network roundtrip to begin tracking your page.
```

Comment:
maybe say "The minimal components of the script are loaded syncrhonously. Full is loaded asynchronously and does not block to begin tracking your page"